ramalama/modelscope.py

```diff
@@ -265,13 +265,12 @@ def _pull_with_model_store(self, args, debug: bool = False):
             return self.store.get_snapshot_file_path(hash, name)
 
         try:
-            # Fetch the SHA-256 checksum of model from the API and use as snapshot hash
-            snapshot_hash = f"sha256:{fetch_checksum_from_api(organization, name)}"
             if not args.quiet:
                 self.print_pull_message(f"ms://{name}:{tag}")
 
-            ms_repo = ModelScopeRepository(name, organization)
-            files = ms_repo.get_file_list(cached_files, snapshot_hash)
+            ms_repo = ModelScopeRepository(name, organization, tag)
```

Comment:
The `tag` isn't used in `ModelSopeRepository` and the `__init__` from its parent class doesn't expect it. So the tag parameter could be removed:
```suggestion
            ms_repo = ModelScopeRepository(name, organization)
```

Or re-add the `__init__` function on ModelScopeRepository so super() can be called with only name and organization.